crates/spirv-builder/src/test/mod.rs

```diff
@@ -49,7 +49,7 @@ spirv-std = { path = "../../crates/spirv-std" }
 "#;
 
 static SRC_PREFIX: &str = r#"#![no_std]
-#![feature(register_attr, asm)]
+#![feature(register_attr, asm, dropck_eyepatch, ptr_internals)]
```

Comment:
why is `dropck_eyepatch` added?

Reply:
When I initially made the test, dropck_eyepatch was required when instantiating the Unique dangling pointer at runtime. I think when I shifted to creating the pointer during the compile as a result of direction from Discord that it must have no longer been needed. I will remove it